packages/mongodb-rag-core/src/contentStore/updateEmbeddedContent.ts

```diff
@@ -39,15 +56,47 @@ export const updateEmbeddedContent = async ({
   sourceNames?: string[];
   concurrencyOptions?: EmbedConcurrencyOptions;
 }): Promise<void> => {
-  const changedPages = await pageStore.loadPages({
+  // find all pages with changed content
+  const pagesWithChangedContent = await pageStore.loadPages({
     updated: since,
     sources: sourceNames,
   });
   logger.info(
-    `Found ${changedPages.length} changed pages since ${since}${
+    `Found ${pagesWithChangedContent.length} changed pages since ${since}${
+      sourceNames ? ` in sources: ${sourceNames.join(", ")}` : ""
+    }`
+  );
+  // find all pages with embeddings created using chunkingHashes different from the current chunkingHash
+  const chunkAlgoHashes = new Map<string, string>();
+  const chunkAlgoHash = getHashForFunc(
+    chunkAlgoHashes,
+    chunkPage,
+    chunkOptions
+  );
```

Comment:
I'm not sure I understand the purpose of the `chunkAlgoHashes` caching here.

* We define an empty cache and only use it for this one call right after.
* That call only gets the current `chunkPage` function (that we import into this file)

It seems like the cache will never actually be used? Am I missing something?